quill-jdbc/src/main/scala/io/getquill/sources/jdbc/JdbcSource.scala

```diff
@@ -67,25 +65,34 @@ class JdbcSource[D <: SqlIdiom, N <: NamingStrategy](config: JdbcSourceConfig[D,
       }
     }
 
-  def execute(sql: String): Int = {
+  def execute(sql: String, generated: Option[String]): Long = {
```

Comment:
@gustavoamigo  Sorry for the late comment on this. I just migrated our code to quill 0.5 and stumbled upon this. I was thinking if generated paramater could have a default value of `None`:  `generated: Option[String] =  None`.

As I user, I used execute to run just plain queries in my database (schema creation). I fixed just adding None to method calls, but I thought about it and maybe the user shouldn't have to mind about adding this None.

What do you think?